modules/smtp/site.js

```diff
@@ -419,7 +421,33 @@ $(function() {
             $('#send_uploaded_files').val(uploaded_files);
             Hm_Ajax.show_loading_icon(); $('.smtp_send').addClass('disabled_input');
             $('.smtp_send_archive').addClass('disabled_input'); 
-            $('.smtp_send').on("click", function() { return false; }); 
+            $('.smtp_send').on("click", function() { return false; });
+
+            if(Hm_Utils.get_from_local_storage('dont_warn_empty_body')) {
+                e.target.submit();
+                return;
+            }
+
+            const modalContent = `
+            <p>You're about to send an empty body</p>
```

Comment:
There should be a message somewhere about an empty subject.

Reply:
Empty subject or empty body you mean? I'm not following.

Reply:
Both. Cypht should warn if subject or body is empty.

Reply:
Good. Implemented✅